External/FEXCore/Source/Interface/Core/ArchHelpers/CodeEmitter/ALUOps.inl

```diff
@@ -10,6 +10,16 @@
  * FEX-Emu ALU operations usually have a 32-bit or 64-bit operating size encoded in the IR operation,
  * This allows FEX to use a single helper function which decodes to both handlers.
  */
+private:
+  bool IsADRRange(int64_t Imm) {
```

Comment:
seems like these can be const qualified or made static

Reply:
You're right, made static.